### PR TITLE
chore(web): update web README.md and local env

### PR DESCRIPTION
**Pull request by mulengawilfred**
# Overview
Not a complete update. If there is anything that can be included please add a comment. Would be good to add as much information so that new members ( sometimes even old members ) can know how to run the project and anything else they would need to know. 
## Summary by CodeRabbit

- **New Features**
	- Introduced new environment configuration variables to enhance the application's setup, including developer mode, API endpoints, and authentication settings.

- **Chores**
	- Updated the `.gitignore` to specifically ignore the `.env` file, allowing for more precise management of environment configurations.
	- Removed the `developerMode` property from the configuration, streamlining settings for production use.

### CODE REVIEW

File: web/.env.local [hunk 1]
```diff
@@ -0,0 +1,7 @@
+REEARTH_WEB_DEVELOPER_MODE=
```
Comment (airslice) [inline web/.env.local:1]:
```suggestion
```
I think this one is only for show create beta project button on dashboard, which is not exists any more, could you help double confirm whether it is still useful?

File: web/.env.local [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+REEARTH_WEB_CESIUM_ION_TOKEN_URL=
```
Comment (airslice) [inline web/.env.local:2]:
```suggestion
```
This one is more for us only, it's not needed for OSS i think.

File: web/.env.local [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+REEARTH_WEB_API=
+REEARTH_WEB_PLUGINS=
+REEARTH_WEB_AUTH0_DOMAIN=
+REEARTH_WEB_AUTH0_AUDIENCE=
+REEARTH_WEB_AUTH0_CLIENT_ID=
```
Comment (airslice) [inline web/.env.local:7]: Maybe we can provide the local urls as well as the template?

File: web/.env.local (hunk 1 above)
Comment (mulengawilfred) [reply web/.env.local:1]: Looks like this one is used in `e2e/utils/config.ts`

File: web/.env.local (hunk 3 above)
Comment (mulengawilfred) [reply web/.env.local:7]: Yes we can. Please share them or push directly to the PR.

File: web/.env.local (hunk 2 above)
Comment (mulengawilfred) [reply web/.env.local]: Okay, I have indicated which ones are required and which ones are optional

File: web/.env.local (hunk 1 above)
Comment (airslice) [reply web/.env.local]: I think it's been configed but not in use any more, i removed it.

File: web/.env.local (hunk 3 above)
Comment (airslice) [reply web/.env.local]: I add the url for first two, the rest can be added after we fix the local auth system i think

**Pull request merged**